manager/service.go

```diff
@@ -44,6 +46,39 @@ var (
 	ErrFailedToCalculateHash = errors.New("error while calculating the hash of the computation")
 )
 
+type Vmpl struct {
+	Value uint32 `json:"value"`
+}
+
+type SnpPolicy struct {
```

Comment:
why not use https://github.com/google/go-sev-guest/blob/00148f763a981a74331226d462983bd9e9b903c4/proto/check/check.pb.go#L44 since it is already defined